main/webapp/modules/core/langs/translation-en.json

```diff
@@ -167,6 +167,7 @@
     "core-index-parser/click-xml": "Click on the first XML element corresponding to the first record to load.",
     "core-index-parser/column-names-label": "Column names (comma separated)",
     "core-index-parser/column-names-optional": "Comma separated",
+    "core-index-parser/update-format-failed": "Update format failed",
```

Comment:
As a user I would find this message quite confusing. (Is it actually grammatical?)
In which conditions will this appear and can we make it a bit more explicit?

Reply:
The error message will appear when an api call with update-format-and-options sub-command fails and yes the message can be made more explicit. Working on it